packaging/debian/changelog

```diff
@@ -0,0 +1,5 @@
+sscg (0-1) unstable; urgency=low
+
+  * Initial release. (Closes: #XXXXXX)
```

Comment:
This will be replaced with an actual bug number once we file an [intent to package](https://wiki.debian.org/ITP) bug.

Reply:
I [filed the ITP](https://bugs.debian.org/986830), and updated this PR with the number.